lib/connect.js

```diff
@@ -27,13 +27,8 @@ const tokenAPI = state =>
 	retry(
 		() =>
 			new Promise((resolve, reject) => {
-				request({
-					uri: 'https://release-auth.zeit.sh',
-					qs: {
-						state
-					},
-					json: true
-				})
+				const qs = queryString.stringify({state});
+				fetch(`https://release-auth.zeit.sh/?${qs}`)
```

Comment:
It looks like we might need to update to call `await res.json()` to get the token on [line 41](https://github.com/zeit/release/pull/161/files#diff-90df8a4fdd65da9a7dabb2c41927a14dR41) with this change

Reply:
@ijjk Nice catch!